### ISSUE TITLE
Undo / redo does not work across tabs

### ISSUE THREAD

**Issue by tim-krieger** [Mindmapp/mindmapp#82]
Steps to reproduce:
1. Create a mind map »test1« with two branches.
2. Create a second mind map »test2« with three branches.
3. Change to »test1« and undo the last changes.

Result:
»test1« is renamed to »test2« (with three branches).

Chromium (Linux), Safari (iOS)

**Comment by cedoor** [comment]
Yes I know it, I have to correctly set mmp to create multiple instances, one for each tab. At the moment the instance is the same and the tabs are managed with the mmp history. Thanks!
